report_txt/models/report.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, api, _
+from odoo.exceptions import UserError
+
+
+class Report(models.Model):
+    _inherit = "report"
+
+    @api.model
+    def _get_report_from_name(self, report_name):
+        reports = self.env['ir.actions.report.xml'].search([
```

Comment:
We need this overwrite, cf the original code: https://github.com/odoo/odoo/blob/10.0/addons/report/models/report.py#L493

And I don't like how it's done in report_xml: https://github.com/OCA/reporting-engine/blob/8.0/report_xml/models.py#L52
I prefer not to call super() and have a code that will be good for everyone, independantly of the reporting engines installed.